mobileapp/www/js/stati.js

```diff
@@ -48,7 +48,7 @@ app.pauseEventHandler = function pauseEventHandler () {
 app.aboutView = function _aboutView () {
   var header = 'About Kloak';
   var logos = '<p><img class="app-logo" src="img/spideroak_logo.png" /> </p>';
-  var info = 'Kloak is an <strong>*experiment*</strong> in social networking that is un-dataminable. All data sent to the server is "end to end" encrypted and unreadable by the server operator. <br /> Kloak is in beta and <strong>should not be used to hide communications from well-equipped potential attackers</strong> <p><a onclick="window.open(\'https://github.com/Crypton/statusapp\', \'_system\')" class="media-link media-link-url">Kloak issue tracker</a> <br />Kloak is built with <a onclick="window.open(\'https://crypton.io\', \'_system\')" class="media-link media-link-url">Crypton</a> by <a onclick="window.open(\'https://spideroak.com\', \'_system\')" class="media-link media-link-url">SpiderOak</a></p>';
+  var info = '<span class="dist-info">Version: ' + app.kloak_version +'<br>Build: ' + app.build + '</span>Kloak is an <strong>*experiment*</strong> in social networking that is un-dataminable. All data sent to the server is "end to end" encrypted and unreadable by the server operator. <br /> Kloak is in beta and <strong>should not be used to hide communications from well-equipped potential attackers</strong> <p><a onclick="window.open(\'https://github.com/Crypton/statusapp\', \'_system\')" class="media-link media-link-url">Kloak issue tracker</a> <br />Kloak is built with <a onclick="window.open(\'https://crypton.io\', \'_system\')" class="media-link media-link-url">Crypton</a> by <a onclick="window.open(\'https://spideroak.com\', \'_system\')" class="media-link media-link-url">SpiderOak</a></p>';
```

Comment:
we should change app.kloak_version to app.app_version as well as add a section to the README about the existence of these required properties that should be in configuration.js
perhaps the best thing to do is add a section for "Configuration" in the readme where you display a sample configuration.js that has comments for each line?